cards/eye-ddx/card.md

```diff
@@ -0,0 +1,33 @@
+---
+
+title: Rule of 2s for Ocular Emergencies

```

Comment:
@jvoros Looks like we may need to revise your tutorial to prevent the `^M` line endings from showing up when copying any pasting from windows machines (which use CRLF line endings, rather than LF)